### ISSUE TITLE
Support reply_to

### ISSUE THREAD

**Issue by billzhong** [sunscrapers/django-templated-mail#1]
Hi,

I have following code:

```python
EmailMessage(self.request, {
    'title': 'test',
}, 'email/message.html').send(
    to=['test1@example.com'],
    reply_to=['test2@example.com'],
)
```

Returns `TypeError: send() got an unexpected keyword argument 'reply_to'`.

Thanks.

**Comment by pszpetkowski** [comment]
Fixed with 1bca21664d93a7995988f715f163ff10ffc51bfb - would be great if you could verify.

**Comment by pszpetkowski** [comment]
New version has been released and I believe this problem has been solved. I'm going to close the issue, however in case of any problems, you're welcome to re-open/create a new one. Thanks.
